keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -190,21 +194,52 @@ internal object ClusterHandlerTest : JUnit5Minutests {
         }
       }
 
-      test("annealing a diff clones the current server group") {
-        whenever(orcaService.orchestrate(any())) doReturn CompletableDeferred(TaskRefResponse("/tasks/1"))
+      context("the diff is only in capacity") {
 
-        converge(request.metadata.name, spec)
+        val diff = javers.compare(resource.spec.withDoubleCapacity(), resource.spec)
 
-        argumentCaptor<OrchestrationRequest>().apply {
-          verify(orcaService).orchestrate(capture())
+        test("annealing resizes the current server group") {
```

Comment:
Tests that if the only thing that changed was capacity we do a "resizeServerGroup"